sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTest.java

```diff
@@ -426,4 +475,29 @@ private DeletedKey pollOnKeyPurge(String keyName) {
         return null;
     }
 
+    /**
+     * Returns a stream of arguments that includes all combinations of eligible {@link HttpClient HttpClients} and
+     * service versions that should be tested.
+     *
+     * @return A stream of HttpClient and service version combinations to test.
+     */
+    static Stream<Arguments> getTestParameters() {
```

Comment:
This method is a duplicate of `getTestParameters()` method in `KeyClientTestBase`.

Reply:
Removed.